ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -316,7 +317,7 @@ forkBlockProduction IS{..} =
                                        currentSlot
                                        (castPoint prevPoint)
                                        prevNo
-                                       (Foldable.toList txs)
+                                       (Foldable.toList . (fmap fst) $ txs)
```

Comment:
Reminder that we can drop this `Foldable.toList` once we switch `getTxs` to return a list (since the type wouldn't catch that its redundant).